src/ProductTour/ProductTour.test.jsx

```diff
@@ -60,11 +60,11 @@ describe('<ProductTour />', () => {
       {
         body: 'Checkpoint 2',
         target: '#target-2',
+        onBack: customOnBack,
       },
       {
         body: 'Checkpoint 3',
         target: '#target-3',
-        onBack: customOnBack,
```

Comment:
Just curious as to why this is being removed from the "Checkpoint 3" test data here.